designs/virtualization/4.x/create-vm-dropdowns/create-vm-dropdowns.md

```diff
@@ -0,0 +1,112 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# create VM dropdowns
+
+This item will cover various updates to the VM creation wizard
+- Adding descriptions to the dropdown fields (using this pattern https://github.com/patternfly/patternfly-design-kit/issues/218)
+- Renaming fields
+- Reordering fields
+
+
+![Create VM - General-default](img/1-0.jpg)
+Landing on the General step, Default mode.
+Section titles added, and fields reorgenized and renamed.
+'General' section:
+- 'Name'
+- 'Description'
+'Resources & configuration' section'
```

Comment:
@suomiy 
In this design, selecting the common template flow hadn't changed at all.
The user selects the OS first, then the "Configuration" (this was "workload profile", I'm not that sure if we should change it now to Configuration, how about "Workload type?"), then the Flavor.
As I understand from Tomas, We've put an effort to make sure that these selections will always result in one option, the selected common template. 
Technically, the user can change the order of selection, though I don't see it likely.

As I understand and see it, the common template is not really for the user, but for our back end. I'm not sure how "centos6-server-small template" can benefit the user unless this info helps him in other steps.

Reply:
> @suomiy
> In this design, selecting the common template flow hadn't changed at all.

I see, can you please add the template field to the designs as well so we can see it altogether?

> The user selects the OS first, then the "Configuration" (this was "workload profile", I'm not that sure if we should change it now to Configuration, how about "Workload type?"), then the Flavor.
> As I understand from Tomas, We've put an effort to make sure that these selections will always result in one option, the selected common template.
> Technically, the user can change the order of selection, though I don't see it likely.

When everything works well then yes. But it is very useful when something goes wrong. For example you have multiple versions of templates on your system and are unsure which one is used.

Also when you select custom flavor - you might be unsure which flavor got actually selected. 

There are many subtleties where knowing which template get used might be useful 

> 
> As I understand and see it, the common template is not really for the user, but for our back end. I'm not sure how "centos6-server-small template" can benefit the user unless this info helps him in other steps.

It might help him to understand why he sees os/cpu/memory limitations.

Reply:
Adding the Common template field as mentioned on a similar comment thread above